code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -373,6 +369,13 @@
 	item_path = /obj/item/guardiancreator/choose/wizard
 	category = "Assistance"
 
+/datum/spellbook_entry/item/holocarp
+	name = "Holocarp Fishsticks"
+	desc = "A tasty treat of fishsticks. On consumption it will grand the user a holocarp guardian spirit, similar to regular guardian spirits. However it is totally possible to have multiple \

```

Comment:
It is a grand error to use `grand` instead of `grant`.

You use the word `multiple` multiple times.

Don't be a parrot, spell it correctly: `tarot`.

Also no whitespace at the end of the whole string. No pun for this one. Sorry.